README.md

```diff
@@ -32,17 +36,17 @@ Scheduler wall time statistics are now disabled by default to keep in line with
 
 That's because you should use tags for stable versions instead! The changelog below should let you know what to expect.
 
-## CHANGELOG ##
```

Comment:
We can write that, no problem. We were waiting to see if the changes we made are desirable and are going to end up being merged in the future :)